src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/impl/CollectionExerciseServiceImpl.java

```diff
@@ -270,21 +270,72 @@ private CollectionExercise newCollectionExerciseFromDTO(CollectionExerciseDTO co
      */
     private void createActionPlans(CollectionExercise collectionExercise, SurveyDTO survey) {
-        log.debug("Creating action plans, CollectionExerciseId: {}", collectionExercise.getId());
-        createOverrideActionPlan(collectionExercise, survey,  "B");
-        createOverrideActionPlan(collectionExercise, survey,  "BI");
-        log.debug("Successfully created action plans, CollectionExerciseId: {}", collectionExercise.getId());
+        log.debug("Creating action plans for exercise, CollectionExerciseId: {}, SurveyId: {}",
+                collectionExercise.getId(), survey.getId());
+        createDefaultActionPlan(survey, "B");
+        createDefaultActionPlan(survey, "BI");
+        createOverrideActionPlan(collectionExercise, survey, "B");
+        createOverrideActionPlan(collectionExercise, survey, "BI");
+        log.debug("Successfully created action plans for exercise, CollectionExerciseId: {}, SurveyID: {}",
+                collectionExercise.getId(), survey.getId());
+    }
+
+    /**
+     * Create default action plan for collection exercise and case type if not already exists
+     * @param survey DTO Representation of survey
+     * @param sampleUnitType Sample Unit Type i.e. (B, H, HI)
+     */
+    private void createDefaultActionPlan(SurveyDTO survey, String sampleUnitType) {
+        log.debug("Creating default action plan, SurveyId: {} , SampleUnitType: {}",
+                survey.getId(), sampleUnitType);
+
+        // If a casetypedefault already exists for this survey/sampleUnitType do nothing
+        CaseTypeDefault existingCaseTypeDefault = caseTypeDefaultRepo.findTopBySurveyIdAndSampleUnitTypeFK(
+                UUID.fromString(survey.getId()), sampleUnitType);
+        if (existingCaseTypeDefault != null) {
+            log.debug("Default action plan already exists, SurveyId: {} SampleUnitType: {}",
+                    survey.getId(), sampleUnitType);
+            return;
+        }
+
+        // Create new action plan and associated casetypedefault
+        String shortName = survey.getShortName();
+        String name = String.format("%s %s", shortName, sampleUnitType);
+        String description = String.format("%s %s Case", shortName, sampleUnitType);
+        ActionPlanDTO actionPlan = actionSvcClient.createActionPlan(name, description);
+        createCaseTypeDefault(survey, sampleUnitType, actionPlan);
+        log.debug("Successfully created default action plan, ActionPlanId: {}, SurveyId: {}, SampleUnitType: {}",
+                actionPlan.getId(), survey.getId(), sampleUnitType);
+    }
+
+    /**
+     * Create case type default for action plan, survey and sample unit type if not already exists
+     * @param survey DTO Representation of survey
+     * @param sampleUnitType Sample Unit Type
+     * @param actionPlan DTO Representation of action plan
+     */
+    private void createCaseTypeDefault(SurveyDTO survey, String sampleUnitType, ActionPlanDTO actionPlan) {
```

Comment:
If this must logically happen on every call to `createDefaultActionPlan`, I don't think it makes sense to have this as a separate method. Nesting method calls like this hurts readability

Reply:
Will look into this. If there's no other situation we're going to want to `createCaseTypeDefaults(/Overrides)` that probably makes sense